plat/renesas/rcar/bl2_plat_setup.c

```diff
@@ -85,6 +86,32 @@ CASSERT((PARAMS_BASE + sizeof(bl2_to_bl31_params_mem_t) + 0x100)
 
 static meminfo_t bl2_tzram_layout __aligned(CACHE_WRITEBACK_GRANULE);
 
+/* FDT with DRAM configuration */
+static uint64_t fdt_blob[4096 / sizeof(uint64_t)];
```

Comment:
4096? Is this meant to be the page size, or is it just an arbitrary number?

Reply:
Arbitrary number into which the DT fits. What better size than a page-aligned one.

Reply:
why not then use PAGE_SIZE_4KB?